system-addon/content-src/lib/detect-user-session-start.js

```diff
@@ -7,7 +7,7 @@ const VISIBILITY_CHANGE_EVENT = "visibilitychange";
 module.exports = class DetectUserSessionStart {
   constructor(options = {}) {
     // Overrides for testing
-    this.sendAsyncMessage = options.sendAsyncMessage || global.sendAsyncMessage;
+    this._store = options.store;
```

Comment:
Looks like `options.store` isn't really optional. Probably have `store` be the first parameter and document it. Also, move this line before the `// Overrides for testing`.